contracts/axelar-gateway/src/contract.rs

```diff
@@ -173,13 +173,18 @@ impl AxelarGatewayInterface for AxelarGateway {
         env: Env,
         signers: axelar_soroban_interfaces::types::WeightedSigners,
         proof: axelar_soroban_interfaces::types::Proof,
+        enforce_rotation_delay: bool,
```

Comment:
Nice, I guess this is a bit better than having to pass in the caller everytime. It does allow the operator to also enforce rotation delay as well so slightly different behaviour than EVM, but makes sense. Add docstring on how this is supposed to be used. It's not clear currently that not just anyone can provide a `false` value

Reply:
I think we need to add docstrings everywhere in the soroban repo. I added as `TODO` in the code, and will add it in a separate PR later.